core/models.py

```diff
@@ -1,6 +1,121 @@
 from django.db import models
+from django.contrib.auth.models import User
 
 # Create your models here.
 
 # TODO: Discuss with team if there is a better way to managing migration and inspectdb at the same time
-from core.models_template import *
+# from core.models_template import *
+
+
+class BaseEntity(models.Model):
+    created_on = models.DateTimeField(auto_now_add=True)
+    updated_on = models.DateTimeField(auto_now=True, blank=True, null=True)
+
+    class Meta:
+        abstract = True
+
+
+class Sector(BaseEntity):
```

Comment:
Could you please add some documentation on some of the less obvious model classes. Just a short description of what it's for.